Enterprise Docs: Caching: Update query caching docs for 8.0 (#34104)
grafana/grafana: docs/sources/enterprise/query-caching.md

- Reduced API costs.
- Reduced likelihood that APIs will rate-limit or throttle requests.

## Enable and configure query caching

> By default, data source queries are not cached. To enable query caching for a single data source, visit the data source settings page, open the Cache tab, and press the Enable button. Once enabled, you can choose a custom TTL for that data source or keep the default TTL.

Author: Done.

> > **Note:** Not all data sources work with query caching. If the Cache tab is not visible in a data source's settings, then the data source does not support query caching.

Reviewer: Try to avoid possessive with inanimate objects such as data sources. It's just weird, and potentially confusing for non-native English readers.

Author: Fair enough :) I gave it a try with a rewording.

To configure global settings for query caching, refer the the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#caching" >}}).

## Disable query caching

To disable query caching for a single data source, visit the data source settings page, open the Cache tab, and press the Disable button.

> To unilaterally disable query caching for all data sources, set the `enabled` flag to `false` in the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#caching" >}}).

Author: This setting disables the query caching _feature_, meaning that no queries will be cached and no users will be able to turn it on or even see the cache settings. I'm trying to describe that.

> To unilaterally disable query caching for all data sources, set the `enabled` flag to `false` in the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#caching" >}}).

Author: Tried a reword.

- Reduced API costs.
- Reduced likelihood that APIs will rate-limit or throttle requests.

> ## Enable and configure query caching

Reviewer: Somewhere we need to call out the required permissions to perform these tasks. For example, you need to be an Organization admin with permissions to edit the data source. Can Team Admins edit these options as well? What about Editors with editors_can_admin enabled?

Reviewer: Looks like this still needs to be added, @mjseaman

Author: Resolved in latest commit.

By default, data source queries are not cached. To enable query caching for a single data source:

> 1. Visit the data source settings page.

Reviewer: Rewrite these top three steps to match the comments in Disable query caching.

Reviewer: 1. On the side menu, click...
2. In the data source list, click the data source that you want to turn off caching for.
3. In the Cache tab, click Disable.

> 1. Open the Cache tab.

Reviewer: ```suggestion
```

> 1. Press the Enable button. 

Reviewer: ```suggestion
1. In the Cache tab, click **Enable**. 
```

> 1. Once enabled, you can choose a custom TTL for that data source or keep the default TTL.

Reviewer: ```suggestion
1. (Optional) Choose a custom TTL for that data source. If you skip this step, then Grafana uses the default TTL.
```

> **Note:** If query caching is enabled and the Cache tab is not visible in a data source's settings, then query caching is not available for that data source.

To configure global settings for query caching, refer the the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#caching" >}}).

## Disable query caching

To disable query caching for a single data source: 

> 1. Visit the data source settings page.

Reviewer: Explain how to do this. 

1. On the side menu, click...
1. In the data source list, click the data source that you want to turn off caching for.
1. In the Cache tab, click **Disable**.

Author: Applied :)

1. Open the Cache tab.

> 1. Press the Disable button.

Reviewer: ```suggestion
1. Click **Disable**.
```

> To disable query caching for an entire Grafana instance, set the `enabled` flag to `false` in the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#caching" >}}). You will no longer see the Cache tab on any data sources and no data source queries will be cached.

Reviewer: ```suggestion
To disable query caching for an entire Grafana instance, set the `enabled` flag to `false` in the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#caching" >}}). You will no longer see the Cache tab on any data sources, and no data source queries will be cached.
```
